linear_algebra/code/CMakeLists.txt

```diff
@@ -66,6 +75,17 @@ target_compile_features(wg21_linear_algebra
         cxx_std_17
 )
 
+target_compile_definitions(wg21_linear_algebra
+    INTERFACE
+        LA_USE_MDSPAN
+#        $<$<NOT:$<CXX_COMPILER_ID:MSVC>>:LA_USE_MDSPAN>
+)
+
+target_link_libraries(wg21_linear_algebra
+    INTERFACE
+        std::mdspan
```

Comment:
This is our transitive dependency upon MDSpan.  By including this as an "INTERFACE" dependency this will make configuration from MDSpan inherit to users of the wg21_linear_algebra target.  They will get the correct include paths, compiler flats, compiler definitions, link flags and any dependencies of MDSpan (none presently).

Now users only have to include wg21_linear_algebra in their target_link_libraries statement and everything will just work.